less/admin/UsersListPage.less

```diff
@@ -59,10 +59,10 @@
       }
 
       &--shaded {
-        background: darken(@body-bg, 3%);
+        background: var(--body-bg-dark);
```

Comment:
I wonder if `darker` might be better than `dark`, since it's not completely dark. Or maybe `shaded` for everything?